Src/World/Timing.cpp

```diff
@@ -1,70 +1,78 @@
 #include "Timing.h"
 
-#include <Math/Math.h>
-
 
 // The maximum amount of time the accumulator can store.
-constexpr double MAX_ACCUMULATED_SECONDS = 3.0;
+static constexpr double MAX_ACCUMULATED_SECONDS = 3.0;
```

Comment:
static is redundant, constexpr implies static.

Reply:
Fixed.
The variable itself will be removed pending changes or updates regarding the timing system.